cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/UserPatchDTO.java

```diff
@@ -0,0 +1,64 @@
+package com.cloudesire.platform.apiclient.dto.model.dto;
+
+public class UserPatchDTO extends DTO
+{
+    private Action action;
+
+    private String provider;
+
+    private Object credentials;
+
+    public UserPatchDTO( Action action )
+    {
+        this.action = action;
+    }
+
+    public UserPatchDTO()
+    {
+    }
+
+    public Action getAction()
+    {
+        return action;
+    }
+
+    public void setAction( Action action )
+    {
+        this.action = action;
+    }
+
+    public String getProvider()
+    {
+        return provider;
+    }
+
+    public void setProvider( String provider )
+    {
+        this.provider = provider;
+    }
+
+    public Object getCredentials()
+    {
+        return credentials;
+    }
+
+    public void setCredentials( Object credentials )
+    {
+        this.credentials = credentials;
+    }
+
+    public enum Action
+    {
+        BLOCK,
+        UNBLOCK,
+        ACTIVATE,
+        APPROVE,
+        SAVE_CLOUD_CREDENTIALS
+    }
+
+    public static class Fields
+    {
+        public static String PROVIDER = "provider";
+        public static String CREDENTIALS = "credentials";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static CREDENTIALS" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make CREDENTIALS a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "CREDENTIALS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3008)